designs/enum-shapes.md

```diff
@@ -125,6 +125,41 @@ structure GetFooInput {
 
 Every enum shape MUST define at least one member.
 
+#### enum members always have a value
+
+Enum members that have neither the `@enumValue` nor the `@enumDefault` trait
```

Comment:
Can we require that a default enumValue is added then? Otherwise, you can't reliably use a selector that gets an enumValue from an enum member without knowing the implementation details of the Smithy model implementation.